api/v1beta1/verticadb_webhook.go

```diff
@@ -149,15 +149,6 @@ func (v *VerticaDB) validateImmutableFields(old runtime.Object) field.ErrorList
 			"shardCount cannot change after creation.")
 		allErrs = append(allErrs, err)
 	}
-	// image will only be allowed to change if autoRestartVertica is disabled
```

Comment:
Just would like to understand why this is removed as it seems that I lost track of webhook design.

Reply:
We wanted this rule in there to make sure users follow the documented steps to upgrade.  But now that the operator automates those steps it wasn't necessary anymore.